Source/BSN.Commons.PresentationInfrastructure/Responses/GenericResponseBase.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace BSN.Commons.Responses
+{
+    public class GenericResponseBase<T> : Response where T : class
```

Comment:
Please add comments for this class

Reply:
I resolved it

Reply:
Please describe the main responsibility of this class

The cause of existence of this class must be describe carefully

thanks